src/utils/constants.ts

```diff
@@ -2,4 +2,7 @@
 
 export const Zero = 0;
 export const One = 1;
+export const Two = 2;
```

Comment:
Heyo, are you still using these? I'd like to avoid adding more to this if I can.

Reply:
Removed the unused. 
 
In terms of "avoid adding more", is there a preferred way/place to define such constants? 
Should we simply disable `@typescript-eslint/no-magic-numbers` where magic numbers are needed?